### DISCUSSION TITLE
Energie-Verbrauch Wärmepumpe aus Zählerkaskade

### DISCUSSION THREAD

**Discussion by unclej84** [solectrus/solectrus#3652] [Q&A] [answered]
Gibt es aktuell schon eine Möglichkeit aus einer Zählerkaskade (Hausanschlusszähler + Hausverbrauchszähler) den Verbrauch z.B. einer zwischen den beiden Zählern abgehenden Wärmepumpe zu ermitteln und als Verbraucher darzustellen? Oder bräuchte man dazu etwas vergleichbares zum Powersplitter, was die Reading der beiden Zähler verrechnet (sowohl Bezug als auch Einspeisung für den PV-Anteil)?

**Comment by ledermann** [comment, marked as answer]
SOLECTRUS benötigt den Verbrauch der Wärmepumpe in einem eigenen InfluxDB-Feld. Den dorthin zu bringen, ist Aufgabe des MQTT-Collectors. Dieser kann auch [rechnen](https://docs.solectrus.de/referenz/mqtt-collector/topics.html#5-mapping-mit-formel), aber nur innerhalb eines Topics.

Was du also benötigst, wäre irgendeine Instanz, die die Berechnung **außerhalb** von SOLECTRUS vornimmt und das Ergebnis (also die Differenz) per MQTT bereitstellt. Möglicherweise geht das in HomeAssistant oder ähnlichen Tools.
